.github/bin/build-and-test.sh

```diff
@@ -111,6 +111,12 @@ case "$MODE" in
     $(dirname $0)/smoke-test.sh
     ;;
 
+  smoke-test-analyzer)
```

Comment:
TODO(future): this is currently the longest running single item in the CI, which will slow down rapid cycles if one wants to wait for 'green'.
Given that we only need the analyzer when actually working on fixing some categories, it might be ok to have this CI item only run on-demand; typically I'd expect someone to run this script manual locally.